From 1aee8c58adfd4a45185b43c2f6b0f3393519bf20 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Wed, 24 Sep 2025 21:52:11 +0000
Subject: [PATCH] Fixing duplicate error in the dtsi table based on 

arch/arm64/boot/dts/vendor/qcom/qrb5165-iot-rb5.dtb: ERROR
(duplicate_label): /soc/i2c@884000/fsa4480@43: Duplicate label 'fsa4480'
on /soc/i2c@884000/fsa4480@43 and /soc/i2c@884000/fsa4480@42

---
 .../arch/arm64/boot/dts/vendor/qcom/kona-v2.1-iot-rb5.dtsi      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/vendor/qcom/kona-v2.1-iot-rb5.dtsi b/arch/arm64/boot/dts/vendor/qcom/kona-v2.1-iot-rb5.dtsi
index 611f32ec87..dc97608204 100644
--- a/arch/arm64/boot/dts/vendor/qcom/kona-v2.1-iot-rb5.dtsi
+++ b/arch/arm64/boot/dts/vendor/qcom/kona-v2.1-iot-rb5.dtsi
@@ -394,7 +394,7 @@

        #include "smb1390.dtsi"

-       fsa4480: fsa4480@43 {
+       fsa4480_43: fsa4480@43 {
                status = "disabled";
        };

--
GitLab

